File: TTs/TT02/Trabalho_Teórico_02/09_cometa/cometa.java
public class cometa {
    public static void main(String[] args){
        int ano = MyIO.readInt();
        while(ano != 0){       
            if((ano - 1986) % 76 == 0) System.out.println(ano + 76);
            else{
                int aux = 2062;
                while(aux < ano) aux += 76;
                System.out.println(aux);
            }
            ano = MyIO.readInt();
        }
    }
}

File: TTs/TT02/Trabalho_Teórico_02/10_cometa_c/cometa.c
/* Bibliotecas Usadas*/
#include <stdlib.h>
#include <stdio.h>

int main(){
    int ano;
    scanf("%d", &ano);
    
    while(ano != 0){
        if((ano - 1986) % 76 == 0) printf("%d\n", ano + 76);
        else{
            int aux = 2062;
            while(aux < ano) aux += 76;
            printf("%d\n",aux);
        }

        scanf("%d", &ano);
    }
    return 0;
}